### ISSUE TITLE
Incompatible requirements when building on release/5.6

### ISSUE THREAD

**Issue by bbfitr** [apple/swift-format#387]
Building the 5.6 branch using Xcode 13.3 produces this error:

> the executable 'swift-format' requires macos 10.11, but depends on the product 'TSCBasic' which requires macos 10.13; consider changing the executable 'swift-format' to require macos 10.13 or later, or the product 'TSCBasic' to require macos 10.11 or earlier.

**Comment by mluisbrown** [comment]
I'm also having the same issue.

**Comment by soumyamahunt** [comment]
Having the same issue in Xode 13.4.1 as well:
```
the executable 'swift-format' requires macos 10.11, but depends on the product 'TSCBasic' which requires macos 10.13; consider changing the executable 'swift-format' to require macos 10.13 or later, or the product 'TSCBasic' to require macos 10.11 or earlier.
```
Any workaround for this?

**Comment by kylebrowning** [comment]
I was able to resolve this by by setting the environment variable `SWIFTTSC_MACOS_DEPLOYMENT_TARGET` to `10.11` no idea if it breaks anything, but it appears to be working

```bash
#!/usr/bin/env bash
# fail if any commands fails
set -ex
export SWIFTTSC_MACOS_DEPLOYMENT_TARGET=10.11

if [ -d "$(pwd)/swift-format/.build" ] 
then
    echo "Linter exists in cache, using it" 
    cd swift-format
else
    echo "Couldn't find linter, cloning, then building"
    git clone https://github.com/apple/swift-format.git
    cd swift-format
    git checkout 'tags/0.50600.0'
    swift build -c release
fi
envman add --key LINTER_BUILD_PATH --value "$(pwd)/.build"
./.build/release/swift-format lint -ps $BITRISE_SOURCE_DIR/App-r
.....
```

**Comment by soumyamahunt** [comment]
Thanks @kylebrowning for the workaround, can this be done as part of the package manifest instead? I was looking for an API to set the current process environment variable in swift but couldn't find any.

Also, it doesn't make sense why the requirement was updated when everything works with the older version requirement.

**Comment by kylebrowning** [comment]
I can't speak to any of that. I just found a work around.

**Comment by allevato** [comment]
It looks like the problem was caused by the swift-tools-support-core 0.2.6 release, which increased its deployment target in a minor version bump. They've since released 0.2.7 that reverts it back to the previous value, and I just verified with Xcode 13.3 that the `release/5.6` branch builds with that version. So I think we can close this out now?
